src/editor/api/scene.js

```diff
@@ -189,6 +191,102 @@ const checkIfImagePathIsEmpty = async (sceneId) => {
   }
 };
 
+const saveScreenshot = async (value) => {
```

Comment:
moved screenshot functionality to the scene api file so that the toolbar component didn't call functions from the screenshotmodalcomponent